sdl2.pc.in

```diff
@@ -8,7 +8,7 @@ includedir=@includedir@
 Name: sdl2
 Description: Simple DirectMedia Layer is a cross-platform multimedia library designed to provide low level access to audio, keyboard, mouse, joystick, 3D hardware via OpenGL, and 2D video framebuffer.
 Version: @SDL_VERSION@
-Requires:
+Requires.private:@PKGCONFIG_DEPENDS@
```

Comment:
We need to be careful here, because SDL2 still supports autotools.
I think it's fine to simply set `PKGCONFIG_DEPENDS` to an empty variable.

**configure.ac**
```
PKGCONFIG_DEPENDS=
AC_SUBST(PKGCONFIG_DEPENDS)
```